tests/common/DotNet.cs

```diff
@@ -100,8 +100,6 @@ public static void CompareApps (string old_app, string new_app)
 						return false;
 
 					// There's a lot of TODOs here, those correspond with missing features in .NET and will be removed as those features are implemented
```

Comment:
comment still needed ?

Reply:
Yes, there are more TODOs further down, and in any case I have pending PRs that will eventually remove this line.